os/include/sys/syscall.h

```diff
@@ -113,23 +113,15 @@
 #define SYS_sem_trywait                (__SYS_sem + 3)
 #define SYS_sem_wait                   (__SYS_sem + 4)
 
-#ifdef CONFIG_PRIORITY_INHERITANCE
 #define SYS_sem_setprotocol            (__SYS_sem + 5)
 #define __SYS_named_sem                (__SYS_sem + 6)
```

Comment:
This internal define is unnecessary. Let's use __SYS_sem at next line continousely.